pynamodb/connection/base.py

```diff
@@ -178,7 +179,8 @@ class Connection(object):
     A higher level abstraction over botocore
     """
 
-    def __init__(self, region=None, host=None, session_cls=None):
+    def __init__(self, region=None, host=None, session_cls=None,
+                 request_timeout_seconds=None, max_retry_attempts=None, base_backoff_ms=None):
```

Comment:
should we not initialize those with default values as below?
DEFAULT_TIMEOUT, DEFAULT_MAX_RETRY_ATTEMPTS_EXCEPTION, DEFAULT_BASE_BACKOFF_MS instead of None.